### ISSUE TITLE
[Feature Request] Provide option to package a game's base, update, and dlc as a single XCI file rather than as a bunch of NSPs.

### ISSUE THREAD

**Issue by eyeonus** [CaitSith2/SwitchSDTool#19]
Simply put, when managing my Switch's installed games, it would be much better from a QOL perspective to only have one file to install, rather than multiple files, in order to get all my stuff back onto it. There is at least one game I know that has 57 (**57!**) DLC, meaning that if I want to reinstall everything, I need to install 59 individual NSPs (base game + latest revision + DLC), whereas if they were all packaged into an XCI, I'd have just the one.

The whole reason for archiving in the first place is so you can free up space now without having to tie up your internet to re-install it later, right? Having everything packaged up in a single file would make that re-install process sooooo much easier.

**Comment by ghost** [comment]
.nsp might be the better format, unless .xci installers can handle ticket and title key without just converting everything to standard crypto. No harm in including both though, in case someone would like to load their games as .xci, with SXOS or in an emulator for example.

In the meantime though, if you need to combine contents into a single file, I believe you may be able to use NSC Builder for this, after you've done your backup.

**Comment by eyeonus** [comment]
Yeah, that's what I've been doing, it'd just be nice to be able to do it all with the same tool.
